modules/mobile/app/helpers/mobile/appointments_helper.rb

```diff
@@ -17,7 +17,7 @@ def create_new_appointment(params)
 
     def backfill_location(appointment)
       unless appointment[:clinic].nil?
-        clinic = vaos_mobile_facility_service.get_clinic(station_id: [:location_id], clinic_id: appointment[:clinic])
+        clinic = vaos_mobile_facility_service.get_clinic(appointment[:location_id], appointment[:clinic])
```

Comment:
The get_clinic method changed slightly and no longer uses keywords. Specs didn't catch this because of mocking. The endpoint that uses this is also not actually in use.